resource-sharing-ui/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
Please change the default app name to something more specific.

Reply:
won't do